pkg/cmd/admin/node/listpods.go

```diff
@@ -101,13 +108,23 @@ func (l *ListPodsOptions) handleRESTOutput(nodes []*kapi.Node, printer kprinters
 		}
 		fieldSelector := fields.Set{GetPodHostFieldLabel(node.TypeMeta.APIVersion): node.ObjectMeta.Name}.AsSelector()
 
-		pods, err := l.Options.KubeClient.Core().Pods(metav1.NamespaceAll).List(metav1.ListOptions{LabelSelector: labelSelector.String(), FieldSelector: fieldSelector.String()})
+		pods := &kapiv1.PodList{}
+
+		restClient := l.Options.KubeClient.Core().RESTClient()
```

Comment:
I was thinking more like, why don't you have an external client ado the .List() on that.